impress-org/givewp
# feature: Add "Protip" message to Donor table page

## Description

This PR introduces an enhancement aimed at providing users with additional information about our  Fee Recovery addon, especially those who may not be aware of it. The primary feature is a "ProTip" message that will be displayed on the Donors table page. The message includes a UTM link that, when clicked, will open the add-on's documentation page in a new window.

To ensure a seamless user experience, the PR includes functionality to track and save the user's interaction with the options. This information will be stored in the database under the options table. If a user has already installed a specific add-on, it will be automatically removed from the list of options to display, preventing redundant or unnecessary messages from appearing.

Option:

Fee Recovery addon - 'givewp_donors_fee_recovery_recommendation_dismissed',

## Affects

Donor Table

## Visuals
https://www.loom.com/share/9203a32eb5ee4f128072de7698492a33

## Testing Instructions
- Install and activate Fee Recovery addon
- Activate the addon and visit the Donor table to confirm that the "ProTip" message does not appear.
- Deactivate the addon and view the Donor table to confirm that the message does appear.
- Close the message via the exit icon on the right, visit the Donor table and refresh the page to confirm the message remains closed.

## Pre-review Checklist
